assets/sass/common/shared/o-forms/_o-form.scss

```diff
@@ -444,7 +444,7 @@ only screen and (min-resolution: 2dppx) {
   .custom-checkbox {
     label {
       background: url('../img/ui/forms/checkbox-01@2x.png') no-repeat;
-      background-size: 50px 1055px;
+      background-size: 50px 1155px;
```

Comment:
No. The courage doesn't have this problem. The checkbox-01@2x.png is not updated on Courage.